awx/ui/src/screens/Instances/InstanceList/InstanceList.js

```diff
@@ -135,6 +142,15 @@ function InstanceList() {
                 onExpandAll={expandAll}
                 qsConfig={QS_CONFIG}
                 additionalControls={[
+                  ...(isK8 && me.is_superuser
```

Comment:
minor terminology pick: K8S is short for K + 8 characters (ubernete) + S.  You should probably keep the S.